packages/dd-trace/test/appsec/index.fastify.plugin.spec.js

```diff
@@ -536,6 +520,93 @@ withVersions('fastify', 'fastify', '>=2', version => {
       })
     })
   })
+
+  describe('Suspicious request blocking - multipart', () => {
+    withVersions('fastify', '@fastify/multipart', multipartVersion => {
+      let server, uploadSpy, axios
+
+      before(function () {
+        // @fastify/multipart is not compatible with Fastify 2.x
+        if (semver.intersects(version, '<= 2.x')) {
```

Comment:
I don't know if this is exactly the case here, but to compare versions, we should use the loadedVersion, and not the version. For example, `version` could be `>0.10`, and the loaded version for that will be the latest, maybe `6.0`.

The callback of `withVersions` method returns 3 parameters, `version` (what we are using now), `module` (IIRC, module name), `loadedVersion` (the real version that will be loading).

And other thing here, why are we using `2.x` and not just `2`?
```suggestion
        if (semver.intersects(version, '<= 2')) {
```